scripts/JSRootTree.js

```diff
@@ -1140,14 +1176,17 @@
 
       this.globals.entry = entry; // can be used in any expression
 
+      this.cut.Produce(this.tgtobj);
+
+      if (this.cut.value == 0) return; 
+
       for (var n=0;n<this.ndim;++n)
```

Comment:
I will merge it, but change logic afterwards.
When dumping values, I still want to see entries where cut condition is negative